distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -525,6 +525,29 @@ type Decimal
     negate : Decimal
     negate self = Decimal.Value self.big_decimal.negate
 
+    ## GROUP CONVERSIONS
+       Convert this to an `Integer`.
+
+       > Example
+         Convert `Decimal` 2345 to an `Integer`.
+             d = Decimal.new "2345"
+             d.to_integer
+             # => 2345
+    to_integer : Integer
+    to_integer self = self.big_decimal.toBigInteger
+
+    ## GROUP CONVERSIONS
+       Convert this to a `Float`.
+
+       > Example
+         Convert `Decimal` 23.45 to a `Float`.
+
+             d = Decimal.new "23.45"
+             d.to_float
+             # => 23.45
+    to_float : Integer
+    to_float self = self.big_decimal.doubleValue
```

Comment:
IMO it would be good to say what happens if the Decimal value is too large (IIRC it gets converted to Infinity, right?) - this is not clear and we should clearly state it. Also IMHO in such cases we should be attaching a warning (ideally I'd fail but I know that'd be too annyoing, IMO warning may be a decent compromise between usability and correctness).

Reply:
I know it's rather obvious, but still I'd add a note that the conversion is not precise (because `Float` is not precise).

Reply:
Done -- attached a warning for both. If we are going to always attach a warning for `from_float`, it makes sense to do the same for `to_float`, since it's even *more* likely that there will be a loss of precision in the case of `to_float`.